src/test/java/com/hazelcast/wm/test/WebFilterConfigTest.java

```diff
@@ -88,4 +88,12 @@ public void propertiesOverrideServletFilterConfiguration() throws Exception {
         WebFilterConfig webFilterConfig = WebFilterConfig.create(servletFilterConfig, properties);
         Assert.assertEquals("cookie1", webFilterConfig.getCookieName());
     }
+
+    @Test
+    public void testBooleanConfigValue() {
+        Properties properties = new Properties();
+        properties.put(WebFilterConfig.COOKIE_HTTP_ONLY, true);
+        WebFilterConfig config = WebFilterConfig.create(emptyFilterConfig, properties);
+        Assert.assertEquals(true, config.isCookieHttpOnly());
+    }
```

Comment:
Let's not limit this test's coverage with boolean values.
```suggestion
    public void testNonStringConfigValues() {
        Properties properties = new Properties();
        properties.put(WebFilterConfig.COOKIE_HTTP_ONLY, true);
        properties.put(WebFilterConfig.COOKIE_MAX_AGE, 160);
        WebFilterConfig config = WebFilterConfig.create(emptyFilterConfig, properties);
        Assert.assertEquals(true, config.isCookieHttpOnly());
        Assert.assertEquals(160, config.getCookieMaxAge());
    }
```